lib/govuk_index/presenters/common_fields_presenter.rb

```diff
@@ -1,70 +1,34 @@
 module GovukIndex
   class CommonFieldsPresenter
+    extend MethodBuilder
+
+    add_method :content_id
```

Comment:
I wonder if it would be clearer to name this something like `map_field`? `add_method` makes it sound more generic than it is.

Reply:
Oops, just spotted Mat's comment below which has a more detailed suggestion. Though we did come up with the same name. 🙃

Reply:
I have refactored the naming here to try and make it clearer, plaese let me know if you think the new names works better/still need improvement?